backend/items/location_test.py

```
@@ -0,0 +1,527 @@
+        
+        # Measure search performance
+        start = time.time()
+        results = Location.objects.filter(
+            full_path__icontains="level1_5",
+            user=self.alice
+        )
+        result_count = results.count()
+        query_time = time.time() - start
+        
+        # Should find Level1_5 and its 10 children
+        self.assertEqual(result_count, 11)
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)